src/main/java/duke/ParsedAnswerHandler.java.orig

```diff
@@ -0,0 +1,121 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+<<<<<<< HEAD
+=======
+
+/**
+ * This class receives a ParsedAnswer object and execute commands based on information
+ * from the object.
+ */
+>>>>>>> branch-A-JavaDoc
+
+public class ParsedAnswerHandler {
+    ParsedAnswer pa;
+
+    ParsedAnswerHandler(ParsedAnswer pa) {
+        this.pa = pa;
+    }
+
+    void execute() {
+        String filePath = System.getProperty("user.dir")  + "/data/storage.txt";
+        switch (pa.getCommand()) {
+            case "bye":
+                System.out.println("Have a nice day.");
+                System.exit(0);
+                break;
+
+            case "list":
+                TaskList.list();
+                break;
+
+            case "todo":
+                ToDos td = new ToDos(pa.getDesc());
+                TaskList.add(td);
+                try {
+                    String sb = "T," +
```

Comment:
i think that sb shouldn't be shortened here because it is hard to figure out what it represents